testing/sdk_cmd.py

```diff
@@ -32,3 +33,12 @@ def run_cli(cmd, print_output=True):
         log.error(err)
         raise dcos.errors.DCOSException(err)
     return stdout
+
+
+def convert_output_to_list(output):
```

Comment:
Yeah I thought about that, then I ended up considering the case where the output of `run_cli` is not expected to be a list, in which case the wrap would break.